highfidelity/hifi
# Map oculus touch grip to trigger

This PR makes it so that the oculus grip buttons do the same thing as the trigger.  This way during demos, it won't matter what people press.

To test: 
1. Run this PR w/ Oculus Touch
2. Squeeze the 'grip' button under your middle finger
3. Observe that it does the same thing as the trigger 

https://highfidelity.fogbugz.com/f/cases/1252/Make-Oculus-secondary-grab-button-grab
